docappendertest/docappendertest.go

```diff
@@ -53,8 +52,41 @@ type BulkRequestItemMeta struct {
 // seconds component.
 const TimestampFormat = "2006-01-02T15:04:05.000Z07:00"
 
+type BulkIndexerResponse struct {
+	Took      int                                  `json:"took"`
+	HasErrors bool                                 `json:"errors"`
+	Items     []map[string]BulkIndexerResponseItem `json:"items,omitempty"`
+}
+
+// BulkIndexerResponseItem represents the Elasticsearch response item.
+type BulkIndexerResponseItem struct {
+	Index        string `json:"_index"`
+	DocumentID   string `json:"_id"`
+	Version      int64  `json:"_version"`
+	Result       string `json:"result"`
+	Status       int    `json:"status"`
+	SeqNo        int64  `json:"_seq_no"`
+	PrimTerm     int64  `json:"_primary_term"`
+	FailureStore string `json:"failure_store,omitempty"`
+
+	Shards struct {
+		Total      int `json:"total"`
+		Successful int `json:"successful"`
+		Failed     int `json:"failed"`
+	} `json:"_shards"`
+
+	Error struct {
+		Type   string `json:"type"`
+		Reason string `json:"reason"`
+		Cause  struct {
+			Type   string `json:"type"`
+			Reason string `json:"reason"`
+		} `json:"caused_by"`
+	} `json:"error,omitempty"`
+}
+
 // DecodeBulkRequest decodes a /_bulk request's body, returning the decoded documents and a response body.
-func DecodeBulkRequest(r *http.Request) ([][]byte, esutil.BulkIndexerResponse) {
+func DecodeBulkRequest(r *http.Request) ([][]byte, BulkIndexerResponse) {
```

Comment:
isn't this breaking the public interface? I'm fine with it but we'll expect a change in every project that uses docappender